### PR TITLE
Careers Page for Website

### PR DESCRIPTION
**Pull request by rauljordan**
This PR adds a career opportunity page for the prylabs website

### CODE REVIEW

File: src/app/pages/careers/prysmatic-careers.page.component.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                </p>
+                <p>
+                    The benefits we offer include:
+                </p>
+                <ul class="bullet py-4 mb-10">
+                    <li>
+                        Insurance <span class="font-bold">(Medical, Life, Vision/Dental)</span> options that are built for both individuals and families, including almost full company contributions
+                    </li>
+                    <li>
+                        Competitive salary and bonus
```
Comment (prestonvanloon) [inline src/app/pages/careers/prysmatic-careers.page.component.html]:
```suggestion
                        Competitive compensation
```

File: src/app/pages/careers/prysmatic-careers.page.component.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                        Built up your engineering skills by attending presentations, workshops, and other speaking engagements on behalf of Prysmatic Labs
+                    </li>
+                    <li>
+                        Shipped the real, live 2.0 Ethereum blockchain with proof of stake and sharding to production, marking one of the most exciting milestones in modern Computer Science
+                    </li>
+                    <li>
+                        Mentored future team-members and junior developers, helping them level up technically
+                    </li>
+                    <li>
+                        Identified and prevented large amounts of technical debt in our repositories through solid design principles and large-scale optimization
```
Comment (prestonvanloon) [inline src/app/pages/careers/prysmatic-careers.page.component.html]: I would hope there aren't large amounts of technical debt! Can we rephrase this to something about code health?

File: src/app/pages/careers/prysmatic-careers.page.component.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+                </p>
+                <ul class="bullet py-4 mb-10">
+                    <li>
+                        Built up your engineering skills by attending presentations, workshops, and other speaking engagements on behalf of Prysmatic Labs
+                    </li>
+                    <li>
+                        Shipped the real, live 2.0 Ethereum blockchain with proof of stake and sharding to production, marking one of the most exciting milestones in modern Computer Science
+                    </li>
+                    <li>
+                        Mentored future team-members and junior developers, helping them level up technically
```
Comment (prestonvanloon) [inline src/app/pages/careers/prysmatic-careers.page.component.html]:
```suggestion
                        Mentored future team-members, junior developers, and external contributors; helping them level up technically
```

File: src/app/pages/careers/prysmatic-careers.page.component.html (hunk 2 above)
Comment (rauljordan) [reply src/app/pages/careers/prysmatic-careers.page.component.html]: Good point lol it sounds like it gives the opposite message!
